src/views/Timesheets/index.js

```diff
@@ -168,10 +168,10 @@ const Timesheets = props => {
   if (props.Authentication) {
     const getActiveJobsForUser = (dateIn, dateOut) => {
       let details = {
-        shift_start_datetime: dateIn.toLocaleString(),
-        shift_end_datetime: dateOut.toLocaleString(),
+        shift_start_datetime: dateIn.toISOString(),
+        shift_end_datetime: dateOut.toISOString(),
```

Comment:
Good! This was tested with a computer set to a different locale, right?

Reply:
Yes, it was tested with computers in a US English locale and locales that were previously causing issues (such as Germany).